tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/TestHello.java

```diff
@@ -50,6 +50,15 @@ private static void printHello(final int n) {
         }
     }
 
+    private static void printIntArray(IntArray a) {
+        int firstValue = a.get(0);
+        int secondValue = a.get(1);
+
+        if (a.getSize() > 1) {
+            Debug.printf("First value %d, second value %d\n", firstValue, secondValue);
+        }
+    }
+
```

Comment:
I would suggest to complement this test, or add a second one that prints the value of each element in the input array, and uses the @Parallel annotation, so that to test the indices passed from the `GlobalThreadIdNode`.

e.g., 
```bash
private static void printIntArray(IntArray a) {
        for (@Parallel int i = 0; i < a.getSize(); i++) {
            Debug.printf("value a[%d] = %d\n", i, a.get(i));
        }
    }
```

Ideally, we can extend the test to assess all 3 dimensions.